greenmail-core/src/main/java/com/icegreen/greenmail/imap/commands/SearchCommandParser.java

```diff
@@ -0,0 +1,87 @@
+package com.icegreen.greenmail.imap.commands;
+
+import com.icegreen.greenmail.imap.ImapRequestLineReader;
+import com.icegreen.greenmail.imap.ProtocolException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.mail.search.AndTerm;
+import javax.mail.search.NotTerm;
+import javax.mail.search.SearchTerm;
+
+/**
+ * Created on 10/03/2016.
+ *
+ * @author Reda.Housni-Alaoui
+ */
```

Comment:
Sorry, it was done automatically by Intellij. I reverted to the original comments.